xml/deployment_installation.xml

```diff
@@ -715,26 +741,352 @@
+      <listitem>
+       <para>
+        In VMDK format, for VMware ESXi.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       VHD
+      </term>
+      <listitem>
+       <para>For Microsoft Hyper-V.</para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <sect4>
+     <title>Overview</title>
+     <para>
+      When deploying a cluster node from pre-installed disk images, the setup
+      program never runs. Thus, it is not possible to set values such as time
+      servers, node names, the &rootuser; password for the &admin_node; and so
+      on in the normal way.
+     </para>
+     <para>      
+      Instead, these values must be set using <literal>cloud-init</literal> configuration
+      files, which are placed into a specific subdirectory within an ISO image
+      which is attached to the VM as a secondary disk. The pre-installed
+      operating system on the disk image looks for a medium with the
+      volume label <literal>cidata</literal>, searches that volume for the
+      configuration files, and applies the settings within them.
+     </para>
+     <para>
+      Full details of the cloud-init metadata files can be found in the
+      Configuration chapter of this manual; for more information, see
+      <xref linkend="cloud-init.configuration"/>.
+     </para>
+     <para>
+      As with individual node installation, the &admin_node; must be configured
+      separately and differently from the &worker_node;s.
+     </para>
+    </sect4>
+    <sect4>
+     <title>Procedure</title>
+     <para>
+      The process is very similar for all hypervisors. The examples in this
+      manual use &kvm; running on &sle;.
+     </para>
+     <para>      
+      You must build at least two ISO images with configuration information:
+      one for the &admin_node; and one for the &worker_node;s. If you wish to
+      assign specific host names or customize individual &worker_node;s, then
+      you must create separate ISO images for each node containing the
+      customised files.
+     </para>
+     <para>
+      Additionally, you must make a separate copy of the downloaded VM disk
+      image for each VM. We suggest keeping the original download elsewhere and
+      making a fresh copy for each node, naming it appropriately: for example,
+      <literal>caas-admin</literal>, <literal>caas-master</literal>, 
+      <literal>caas-worker1</literal>, <literal>caas-worker2</literal> and so
+      on.
+     </para>
+     <sect5>
+      <title>Configuration Files</title>
+      <para>
+       There are two separate configuration files: <literal>user-data</literal>
+       and <literal>meta-data</literal>. Each node needs both. Thus, you must at
+       a minimum prepare one pair of files for the &admin_node;, and another
+       pair of files for all the &worker_node;s.
+      </para>
+      <para>
+       The two files must be placed into a subdirectory. For the &admin_node;,
```

Comment:
a lot of must's here and pasive voices, try to avoid them: "You need to put two files into subdirectories. For the admin_node, name the file cc-admin.....etc

Reply:
Fixed.